templates/rule.conf.erb

```diff
@@ -1,4 +1,8 @@
+# File controlled by Puppet
+# DO NOT EDIT LOCALLY
+#
 # <domain> <type> <item> <value>
 #
-
-<%= @ulimit_domain %>		<%= @ulimit_type %>		<%= @ulimit_item %>		<%= @ulimit_value %>
+<% (0..@ulimit_type.length-1).each do |i| %>
```

Comment:
I'm a little puzzled by ```<% (0..@ulimit_type.length-1).each do |i| %>```.
What's the purpose of adding these range limits if you could just iterate the array?